scripts/develop.sh

```diff
@@ -136,7 +137,7 @@ fatal() {
 	trap 'fatal "Script encountered an error"' ERR
 
 	cdroot
-	start_cmd API "" "${CODER_DEV_SHIM}" server --http-address 0.0.0.0:3000 --swagger-enable --access-url "${CODER_DEV_ACCESS_URL}" --dangerous-allow-cors-requests=true "$@"
+	start_cmd API "" "${CODER_DEV_SHIM}" server --http-address 0.0.0.0:3000 --swagger-enable --access-url "${CODER_DEV_ACCESS_URL}" --dangerous-allow-cors-requests=true --external-token-encryption-keys="${EXTERNAL_TOKEN_ENCRYPTION_KEYS}" "$@"
```

Comment:
sanity check: If you enable it this way, will it break existing dev deployments of Coder dev environments? I don't mind removing `.coderv2` dir, but just asking.

Reply:
Last I checked it didn't break anything. If it does break a dev deployment, then that's a serious bug. Enabling it by default might be a bit obnoxious though?

Reply:
SGTM 👍